### PR TITLE
Use angle brackets for non-relative paths

### PR DESCRIPTION
**Pull request by gaida-exe**
Since Ken mentioned that he wants angle brackets, I also created an angle bracket PR and this would be the counterpart to https://github.com/AcademySoftwareFoundation/openvdb/pull/1363. 

It shortens the relative paths that could be shortened and unifies all not relative paths to use angle brackets.

### CODE REVIEW

File: openvdb/openvdb/tools/PotentialFlow.h [hunk 1]
```diff
@@ -17,7 +17,6 @@
 #include "Mask.h" // interiorMask
 #include "Morphology.h" // erodeActiveValues
 #include "PoissonSolver.h"
-#include <openvdb/openvdb.h>
```
Comment (e4lam) [inline openvdb/openvdb/tools/PotentialFlow.h:20]: I haven't taken a close look here but what's the reason for removing this include? I'm a strong proponent for IWYU.

Comment (gaida-exe) [reply openvdb/openvdb/tools/PotentialFlow.h:20]: It's a double include at line [13 and 20](https://github.com/AcademySoftwareFoundation/openvdb/pull/1418/files/978961ca2d11a6d61707596338fca2b3cbff38f7#diff-f3ebcfe3981bcdd8af15efe8f842133e9e98d510e0b6fa5edf42542cfce0f6d1L13-L20).

Comment (e4lam) [reply openvdb/openvdb/tools/PotentialFlow.h:20]: Ah, thank you!

**Review by e4lam — APPROVED** [review]
Generally looks ok to me then!

**Comment by gaida-exe** [comment]
Sorry for the ping, but would be great, if you could run the CI and maybe take a look.
(you took a look at the previous PR https://github.com/AcademySoftwareFoundation/openvdb/pull/1363)
@jmlait @Idclip @danrbailey

**Review by jmlait — APPROVED** [review]

**Comment by gaida-exe** [comment]
I've just pulled master without conflicts, because the CI failed. Maybe this can fix the issues. On Windows TBB was missing.
([failed Action](https://github.com/AcademySoftwareFoundation/openvdb/actions/runs/3169314994/jobs/5287905503#step:5:32))

**Review by kmuseth — APPROVED** [review]
Looks good to me

I have no ides why the NanoVDB unit-tests are failing - it looks like a CI installation issue related to CUDA drivers. In other words, nothing related to this PR

**Comment by kmuseth** [comment]
can you try this in your fork:

```
git fetch upstream
git merge upstream/master
git push origin angle-brackets
```

assuming you previously did 

`git remote add upstream git@github.com:AcademySoftwareFoundation/openvdb.git`

**Comment by gaida-exe** [comment]
Thanks for the suggestion Ken, but the branch is already merged with the latest commit of the upstream OpenVDB version ([0efd583](https://github.com/AcademySoftwareFoundation/openvdb/pull/1418/commits/0efd583dc28695cf5babddee0193e8cbbe310817))
And as said, only the non-related NanoVDB build is failing, because of CUDA.
I don't know if it would change anything, but maybe rerunning the CI without a change could work, if something blocked the install.

**Pull request merged**